### ISSUE TITLE
UI bug

### ISSUE THREAD

**Issue by tristandostaler** [Shuffle/openapi-apps#38]
When I create a custom app and then delete it, I see this on the right in the Apps panel:
![image](https://user-images.githubusercontent.com/5489330/150844043-fd1b2af0-13e9-4c9c-bd19-7a4d2e1f88d0.png)

**Comment by frikky** [comment]
> When I create a custom app and then delete it, I see this on the right in the Apps panel: ![image](https://user-images.githubusercontent.com/5489330/150844043-fd1b2af0-13e9-4c9c-bd19-7a4d2e1f88d0.png)

ooh, haven't seen this before. Wondering whether it's a caching bug that doesn't clean up properly :thinking:

**Comment by tristandostaler** [comment]
Maybe, but not in the browser. I tried Empty Cache and Hard Reload. It might be linked to my other bug where a node executes forever?

**Comment by frikky** [comment]
> Maybe, but not in the browser. I tried Empty Cache and Hard Reload. It might be linked to my other bug where a node executes forever?

Nope, shouldn't be connected. It's most likely a caching issue on the webserver itself. It's only noticeable now, as we ended up changing to see all apps, whether valid and active or not. I suppose this is connected to an old system of apps being saved within each users' object rather as a global entity, and it's unable to delete it somehow. 

Thanks
